cmake/GenerateLayerJson.cmake

```diff
@@ -10,7 +10,7 @@ function(GENERATE_LAYER_JSON_FILE TARGET RELATIVE_PATH_PREFIX LAYER_BINARY IN_FI
 
     set(CONFIG_DEFINES
         -DINPUT_FILE=${IN_FILE}
-        -DVK_VERSION=${VULKAN_VERSION_MAJOR}.${VULKAN_VERSION_MINOR}.${VULKAN_VERSION_PATCH}
+        -DVK_VERSION=${VK_VERSION_STRING}
```

Comment:
Is this an exact-match identical string?

Reply:
Should be

https://github.com/KhronosGroup/Vulkan-Headers/blob/df60f0316899460eeaaefa06d2dd7e4e300c1604/CMakeLists.txt#L35C1-L35C1